Jenkinsfile

```diff
@@ -60,60 +88,60 @@ timestamps() {
 					sh 'npx retire --exitwith 0'
 
 					step([$class: 'WarningsPublisher', canComputeNew: false, canResolveRelativePaths: false, consoleParsers: [[parserName: 'Node Security Project Vulnerabilities'], [parserName: 'RetireJS']], defaultEncoding: '', excludePattern: '', healthy: '', includePattern: '', messagesPattern: '', unHealthy: ''])
-				} // stage
+				} // nodejs
+			} // stage
+		} // node
 
-				stage('Publish') {
+		node('(osx || linux) && git && npm-publish') {
+			stage('Publish') {
+				checkout scm
+				nodejs(nodeJSInstallationName: "node ${nodeVersion}") {
 					if (!isPR) {
```

Comment:
we've generally moved away from publishing on non-PRs to doing a regex sniff of branch names to only publish off say 'master' and \d_\d_\d branches.